apollo-compiler/src/test/graphql/com/example/prepend_getter/TestOperation.graphql

```diff
@@ -0,0 +1,5 @@
+query TestQuery {
+  hero {
+    name
+  }
+}
```

Comment:
Could you please update query to include fragments and inline fragments. I believe there will be some other places where we need to update not just for `Field`

Reply:
Oh, that's a great callout, thanks.